internal/storage/v2/clickhouse/sql/queries.go

```diff
@@ -199,13 +199,13 @@ SELECT
     scope_str_attributes.key,
     scope_str_attributes.value,
     scope_complex_attributes.key,
-    scope_complex_attributes.value,
+    scope_complex_attributes.value
 FROM
-    spans
-WHERE
-    trace_id = ?
+    spans s
```

Comment:
what happens to the ordering here? Since the iterator just packages each loaded span into Traces{} and pushes it up, the query itself must guarantee that spans for the same trace ID always come together, otherwise we will break the contract of Storage API on how the results must be returned.